custom_components/yahoofinance/const.py

```diff
@@ -46,6 +46,13 @@
 
 DATA_REGULAR_MARKET_PREVIOUS_CLOSE: Final = "regularMarketPreviousClose"
 DATA_REGULAR_MARKET_PRICE: Final = "regularMarketPrice"
+DATA_PRE_MARKET_PRICE: Final = "preMarketPrice"
+DATA_POST_MARKET_PRICE: Final = "postMarketPrice"
+DATA_PRE_MARKET_STATE: Final = "PRE"
+DATA_PREPRE_MARKET_STATE: Final = "PREPRE"
+DATA_POST_MARKET_STATE: Final = "POST"
+DATA_POSTPOST_MARKET_STATE: Final = "POSTPOST"
```

Comment:
What do PREPRE and POSTPOST values indicate? Google search is not turning up any info about this.

Can you include sample content for the same under tests ?